io.asgardio.dotnet.oidc.sdk.sample/App.config

```
@@ -0,0 +1,85 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<!--Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+    
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    
+    http://www.apache.org/licenses/LICENSE-2.0
+    
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License. -->
+<configuration>
+  <configSections>
+    <section name="log4net" type="log4net.Config.Log4NetConfigurationSectionHandler,Log4net" />
+  </configSections>
+  <log4net>
+    <appender name="TestAppender" type="log4net.Appender.RollingFileAppender">
+      <file value="E:\log\MyTestAppender.log" />
+      <encoding value="utf-8" />
+      <appendToFile value="true" />
+      <rollingStyle value="Date" />
+      <!--<rollingStyle value="Size" />
+      <maxSizeRollBackups value="5" />
+      <maximumFileSize value="5MB" />
+      <staticLogFileName value="true" />-->
+      <layout type="log4net.Layout.PatternLayout">
+        <conversionPattern value="%date %level [%thread] %type.%method - %message%n" />
+      </layout>
+    </appender>
+    <root>
+      <level value="All" />
+      <!-- If the following line is not included the log file 
+      will not be created even if log4net is configured with this file. -->
+      <appender-ref ref="TestAppender" />
```

```suggestion
      <appender-ref ref="TestAppender" />
```

Instead of saying `Test`. Shall we use a term like `Empty` or else?